boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_m7.dts

```diff
@@ -21,6 +22,18 @@
 		zephyr,console = &uart4;
 		zephyr,shell-uart = &uart4;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		yellow_led: led_0 {
+			gpios = <&gpio3 16 GPIO_ACTIVE_HIGH>;
```

Comment:
May I know hardware connection of GPIO3_16 pin? as I can't find this led on the board i.MX 8M Plus EVK board.